shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/test/java/org/apache/shardingsphere/db/protocol/postgresql/constant/PostgreSQLColumnTypeNameOidMappingsTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.db.protocol.postgresql.constant;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class PostgreSQLColumnTypeNameOidMappingsTest {
+    
+    @Test
+    public void assertIntegerArrayGot() {
+        int columnTypeName = PostgreSQLColumnTypeNameOidMappings.getTypeOidByColumnTypeName("_int4");
+        assertEquals(1007, columnTypeName);
```

Comment:
For single parameter asserts, assertTrue, assertFalse, assertNull and assertNotNull should be used.
For multiple parameter asserts, assertThat should be used.